lib/internal/modules/package_json_reader.js

```diff
@@ -10,30 +14,53 @@ const cache = new SafeMap();
 let manifest;
 
 /**
- *
+ * Returns undefined for all failure cases.
  * @param {string} jsonPath
  */
 function read(jsonPath) {
   if (cache.has(jsonPath)) {
     return cache.get(jsonPath);
   }
 
-  const { 0: string, 1: containsKeys } = internalModuleReadJSON(
+  const {
+    0: includes_keys,
```

Comment:
nit: this is javascript, where camelCase is the standard

```suggestion
    0: includesKeys,
```

Reply:
Definitely, maybe we should have an eslint rule about this in a different PR.

Reply:
I'm surprised there isn't one @aduh95 I do you know if this has been discussed previously?